pipeline/relation_embedder/relation_embedder/scripts/cli.py

```diff
@@ -0,0 +1,29 @@
+import boto3
+import os
+import sys
+import subprocess
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+CLI_MAIN = os.path.join(HERE, "..", "target", "universal", "stage", "bin", "cli-main")
+
+
+def get_secret_string(session, secret_id):
+    """
+    Look up the value of a SecretString in Secrets Manager.
+    """
+    secrets = session.client("secretsmanager")
+    return secrets.get_secret_value(SecretId=secret_id)["SecretString"]
+
+
+def set_environment(pipeline_date):
+    session = boto3.Session()
+    prefix = f"elasticsearch/pipeline_storage_{pipeline_date}/"
+    os.environ["es_host"] = get_secret_string(session, f"{prefix}public_host")
```

Comment:
I think there may be room to have a `.env` file, or some means to generate one from which to load environment variables.  Now we may have a way to avoid referencing secrets directly in environment variables passed to an app i'm more comfortable about using that mechanism.

Not blocking, but related to https://github.com/wellcomecollection/catalogue-pipeline/pull/2785

Reply:
I think that once #2785 is done, there's a good opportunity to harmonise the local and lambda approaches.